symfony/src/Entity/Notice.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\NoticeRepository")
+ * @ORM\Table(name="notices")
+ */
+class Notice
+{
+    /**
+     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="IDENTITY")
```

Comment:
Not sure if this is generated code, but was curious whether we had to specify column names and the `GeneratedValue` strategy for our stuff?

Reply:
I auto-generated this entity using the `doctrine:mapping:import` command. I'll look into whether we should remove the explicit strategy parameter.